dist/view.js

```diff
@@ -298,4 +299,43 @@ function reactVirtualizedToReactVirtualized(view) {
     rowCount: rowCount,
     columnWidth: columnWidth
   };
+}
+
+/**
+  * Ensure dates and times in resources are in ISO format.
+  * @param {resource}
+  * @return {resource} with formatted dates and times
+  */
+function normalizeDateAndTime(resource) {
+  if (resource._values) {
+    var dateFields = [],
```

Comment:
@Mikanebu actually, this is inside `/dist/` which is compiled ES5 so it is `var` everywhere.